website/project/views/contributor.py

```diff
@@ -633,7 +633,6 @@ def claim_user_form(auth, **kwargs):
             # Redirect to CAS and authenticate the user with a verification key.
```

Comment:
Just realized that the helper is for token. I do need new one for verification key. :) @icereval